.circleci/config.yml

```diff
@@ -5,10 +5,10 @@ orbs:
     executors:
       node:
         docker:
-          - image: circleci/node:12
+          - image: circleci/node:14
```

Comment:
Why? All the projects that import this module use node 12, so I feel like this has the opportunity to create some incompatibilities.

Reply:
We should be able to handle any specific backwards-compatibility requirements handled by ensuring the `tsconfig.json` compilation options are set appropriately.

More to the point, though, node 12 is two years old by now. It's already out of active support, and will be completelly EOL next year. Something's gotta be the first thing updated -- and hdwallet has better test coverage than some of its prominent dependents do. (Besides, Node 14 has WASI, and that will probably come in handy if/when there's a WASM engine for the isolated crypto stuff.)